libs/state/selections/spec/select.spec.ts

```diff
@@ -1,16 +1,16 @@
 // eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
+import { select } from '@rx-angular/state/selections';
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
```

Comment:
I think we can use a relative import here and remove the eslint disabled rule.

Reply:
Good catch, fixed.